lib/src/arg_parser.dart

```diff
@@ -307,16 +307,18 @@ class ArgParser {
     return Usage(_optionsAndSeparators, lineLength: usageLineLength).generate();
   }
 
-  /// Get the default value for an option. Useful after parsing to test if the
-  /// user specified something other than the default.
-  dynamic getDefault(String option) {
+  /// Returns the default value for [option].
+  dynamic defaultFor(String option) {
     var value = options[option];
     if (value == null) {
       throw ArgumentError('No option named $option');
     }
     return value.defaultsTo;
   }
 
+  @Deprecated('use defaultFor instead')
```

Comment:
Do we tend to format these like proper sentences? `'Use defaultFor instead.'`? I don't think I've ever actually used `@deprecated` before (because all of my APIs are so perfect they never need to be deprecated, of course).

Reply:
I don' think I've ever thought about it. I'll go ahead and format like sentences.